### ISSUE TITLE
wicked gem

### ISSUE THREAD

**Issue by Nerian** [code-and-effect/effective_datatables#142]
Hi,

I got this error after I updated to effective resources 1.6.5 from 1.6.3:
````
RuntimeError: please install gem 'wicked' to use Effective::WizardController
  config/environment.rb:5:in `<top (required)>'
    Rails.application.initialize!
  bin/bundle:3:in `load'
    load Gem.bin_path('bundler', 'bundle')
  bin/bundle:3:in `<main>'
    load Gem.bin_path('bundler', 'bundle')
````

Strange thing is that I did not get an error during test or development. But app didn't start in prod. Once I added `wicked` to my Gemfile it worked fine. I was wondering why don't we add this gem to the gemspec of effective datatables, since it seems to be a requirement.

**Comment by matt-riemer** [comment]
Hey this was fixed in effective_resources 1.6.6.

Here's the relevant commit

https://github.com/code-and-effect/effective_resources/commit/b13a2682624fea35f0f2acc8712434bbd26f02a9

Sorry, this wicked gem is supposed to be optional, only used when making a wizard controller.

I'm honestly surprised that the code passed CI with no problems, but failed on production. That zeitgeist class loader does unexpected things sometimes, or my test environment doesn't match production as close as I'd like

Cheers, sorry for any inconvenience

**Comment by Nerian** [comment]
Thanks!
